components/definitions/chromel.d.ts

```diff
@@ -18,8 +18,8 @@ declare namespace chrome.braveRewards {
   const getPublisherData: (windowId: number, url: string, faviconUrl: string, publisherBlob: string | undefined) => {}
   const getWalletProperties: () => {}
   const getCurrentReport: () => {}
-  const onWalletCreated: {
-    addListener: (callback: () => void) => void
+  const onWalletInitialized: {
+    addListener: (callback: (result: number) => void) => void
```

Comment:
instead of number, you can use `Rewards.Result`

Reply:
Done